lib/ansible/modules/cloud/amazon/ec2_vpc_nacl.py

```diff
@@ -29,11 +29,19 @@
   name:
     description:
       - Tagged name identifying a network ACL.
-    required: true
+      - One of the name or nacl_id is required.
```

Comment:
Also, use name quoting - in this case, as they're option names, it should be `I(name) and I(nacl_id)`